nbconvert_html5/templates/semantic-forms/smol.html.j2

```diff
@@ -0,0 +1,154 @@
+{%- extends 'semantic-forms/cell.html.j2' -%}
+
+{% from 'mathjax.html.j2' import mathjax %}
+{% from 'jupyter_widgets.html.j2' import jupyter_widgets %}
+
+{% block css %}
+<meta name="color-scheme" content="dark light">
+
+<style>
+    {# use the system ui and a larger font #}
+    body {
+        font-size: larger;
+        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    }
+
+    {# use flex to display the main area. each cell will be arranged according to the flex display, grid could be used instead. using flex generates new interactive elements when horizontal or vertical scroll is required. #}
+
+    /** focus **/
+    section:focus-within {
+        border: 2px solid;
+    }
+
+    section:hover {
+        outline: 2px dashed;
+        outline-offset: -3px; {# border offset plus a half #}
+    }
+
+    /** layout **/
+    fieldset[name=input]:disabled~ul[role=toolbar],
+    form[action=markdown]+section>fieldset[name=input]:disabled,
+    main>section.markdown>fieldset[name=outputs]>legend+label {
+        display: none;
+    }
+
+    main {
+        display: flex;
+        overflow-x: hidden;
+        flex-direction: column;
+    }
+
+    main>section {
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+        gap: 1rem 1rem;
+    }
+
+    main>section>fieldset[name=input],
+    main>section.markdown>fieldset[name=outputs] {
+        border: none;
+        display: flex;
+        flex-direction: row;
+    }
+
+    textarea[name=source], fieldset[name$=put]>output {
+        width: 100%;
+    }
+
+    main>section {
+        overflow-x: auto;
+    }
+
+    /** fix the color constrast on disabled textareas **/
+    fieldset[name=input]:disabled>textarea {
+        color: unset;
+    }
+
+    /** skip links **/
+    .visually-hidden:not(:focus):not(:active) {
+        clip: rect(0 0 0 0);
+        clip-path: inset(50%);
+        height: 1px;
+        overflow: hidden;
+        position: absolute;
+        white-space: nowrap;
+        width: 1px;
+    }
+</style>{% endblock css %}
+{% block any_cell scoped %}
+{%- set count = nb["cells"].index(cell)-%}{%- set ID = "/cells/" + str(count) -%}
+{%- set CODE = cell.cell_type == "code" -%}
+{%- set tags = celltags(cell) -%}
+<form hidden id="{{ID}}" name="{{ID}}" method="POST" action="{{cell.cell_type}}" aria-labelledby={{ID}}/cell_count>
+    <label id={{ID}}/cell_count>Cell&nbsp;<output name="cell_count" role="presentation">{{count + 1}}</output></label>
+</form>
+<section aria-labelledby="{{ID}}/cell_count" class="cell {{cell.cell_type}}{% if tags %} {{tags}}{% endif %}" id="{{count}}">
```

Comment:
i'm not happy. i can't get VoiceOver reliably to announce cells. they are only announced when there are at least to block nodes to be announced.

bundling contiguous markdown content into a contiguous cell will more reliably get VoiceOver to announce leaving the region.